stable/fairwinds-insights/values.yaml

```diff
@@ -208,9 +208,9 @@ openApi:
     # -- Create a horizontal pod autoscaler for the Open API server.
     enabled: false
     # -- Minimum number of replicas for the Open API server.
-    min: 1
+    min: 2
     # -- Maximum number of replicas for the Open API server.
-    max: 2
+    max: 3
```

Comment:
what do you think about making this `1/2` by default?

It's just serving API documentation, so kind of a secondary feature.

Reply:
I believe the issue is that if it's set to 1 at any time, the pdb will cause node rolls to fail by blocking the pod deletion. So I believe we should either remove the pdb completely or increase the hpa's defaults.

Reply:
Makes sense!